lib/views/auth_screen.dart

```diff
@@ -100,9 +99,9 @@ class _AuthScreenState extends State<AuthScreen>
           mainAxisAlignment: MainAxisAlignment.spaceEvenly,
           children: <Widget>[
             Expanded(
-              child: FlatButton(
-                splashColor: Colors.white,
-                highlightColor: Colors.white,
+              child: TextButton(
```

Comment:
i assume this change is because flatbutton is deprecated. But this change doesn't match with the PR. If you're doing this upgrade, pls do it for entire project